package-lock.json

```diff
@@ -60,6 +60,53 @@
         "vitest": "^3.2.4"
       }
     },
+    "../purchases-ui-js": {
```

Comment:
Hmm this is weird... Why did it change to use the npm dependency to use a local dependency?